feature/announcement/src/main/res/layout/item_announcement.xml

```diff
@@ -38,15 +38,15 @@
         <androidx.emoji.widget.EmojiTextView
             android:id="@+id/title_text"
             android:layout_width="0dp"
-            android:layout_height="0dp"
+            android:layout_height="wrap_content"
             android:layout_marginEnd="27dp"
             android:layout_marginStart="16dp"
             android:ellipsize="end"
             android:gravity="start|center_vertical"
             android:maxLines="2"
+            android:minHeight="40dp"
```

Comment:
I guess you can use a Barrier for this purpose like below. Could you please try it?

```
<androidx.constraintlayout.widget.Barrier
   android:id="@+id/titleBottomBarrier"
   ...
   app:barrierDirection="bottom"
   app:constraint_referenced_ids="category_icon,title_text"
   />

<androidx.emoji.widget.EmojiTextView
    android:id="@+id/title_text"
    ...
    app:layout_constraintBottom_toBottomOf="@id/titleBottomBarrier"
    />
```

Reply:
ok! I will try it!